docs/setup/configuration.md

```diff
@@ -36,7 +36,7 @@ The `server` object must be configured as following:
 
 ## Server slave configuration
 
-The `serverSlave` configuration is required when you're running the `server/server-slave.js` application!
+The `serverSlave` configuration is required when you're running the `back-end/module-client.js` application!
```

Comment:
Do you even LIFT BRO

Reply:
Replaced `serverSlave` config key with `moduleClient`